cf7_to_trello/index.js

```diff
@@ -1,71 +1,38 @@
 const express = require('express');
 const app = express();
-const util = require('util');
 const Trello = require('./models/Trello');
 
 // Check environment
-
 if (!process.env.GCP_PROJECT) {
-  console.log("[i] Not in the cloud: entering debug mode")
-  // Set up body parsing middleware
-  const bodyParser = require('body-parser');
-  app.use(bodyParser.urlencoded({ extended: false }));
-  app.use(bodyParser.json());
-
-  const port = process.env.PORT || 9000;
-  app.listen(port, () => {
-    console.log('[i] Service listening on port', port);
-  });
+    console.log("[i] Not in the cloud: entering debug mode");
+    // Set up body parsing middleware
+    const bodyParser = require('body-parser');
+    app.use(bodyParser.urlencoded({extended: false}));
+    app.use(bodyParser.json());
+
+    const port = process.env.PORT || 9000;
+    app.listen(port, () => {
+        console.log('[i] Service listening on port', port);
+    });
 }
 
 // Default route
 app.get('/', (req, res) => {
-  res.send("Yes hello");
+    res.send("Yes hello");
 });
 
-// Get data from CF7 Webhook
+// Get data from CF7 Webhook and create trello card
 app.post('/', async (req, res) => {
-  // let submission = req.body;
-  // submission.phone = sanitizePhoneNr(submission.phone);
-  // create trello card
-  const trello = new Trello();
-  await trello.init();
-  try {
-    console.log('[i] New request from CF7 ' + JSON.stringify(req.body));
-    await trello.newCard(req.body);
-    res.send("Success!\n");
-  } catch (err) {
-    res.send("Failed because of reasons, error has been logged " + err.message);
-  }
+    const trello = new Trello();
+    await trello.init();
+    try {
+        let data = req.body;
```

Comment:
If parameter is not suposed to be changed, it's better to define it as a const.
```suggestion
        const data = req.body;
```